src/components/MoneyRequestHeader.js

```diff
@@ -90,6 +117,7 @@ function MoneyRequestHeader(props) {
                     personalDetails={props.personalDetails}
                     shouldShowBackButton={props.isSmallScreenWidth}
                     onBackButtonPress={() => Navigation.goBack(ROUTES.HOME, false, true)}
+                    statusBar={isScanning ? getStatusBar() : null}
                     shouldShowBorderBottom
```

Comment:
Hmm. This is a new pattern we're building out, and I suspect more screens will want to supply a statusbar. Instead of inlining, maybe we can move this to a separate MoneyRequestHeaderStatusBar component?

Reply:
MoneyRequestHeaderStatusBar looks good